tensorrt_llm/_torch/modules/attention.py

```diff
@@ -15,6 +15,24 @@
 from .rotary_embedding import RotaryEmbedding
 
 
+class L2Norm(torch.nn.Module):
+
+    def __init__(self, dtype: Optional[torch.dtype] = None, eps: float = 1e-6):
+        super().__init__()
+        self.eps = eps
+        self.dtype = dtype
+
+    def _norm(self, x):
+        flash_infer_kernel = RMSNorm(hidden_size=x.shape[-1],
+                                     eps=self.eps,
+                                     dtype=self.dtype,
+                                     device=x.device)
+        return flash_infer_kernel(x)
+
+    def forward(self, x):
+        return self._norm(x.contiguous()).type_as(x)
```

Comment:
```suggestion
        return self._norm(x)
```
flash_infer norm now supports strided inputs.